pages/claims/info.js

```diff
@@ -49,4 +48,4 @@ const mapStateToProps = (state) => {
   return { user, claims };
 };
 
-export default withRedux(configureStore, mapStateToProps, mapDispatchToProps)(showClaimPage);
+export default withRedux(configureStore, mapStateToProps, mapDispatchToProps)(ShowClaimPage);
```

Comment:
So, we never use camelCase for all commands, right?

Reply:
Yes, it is recommended to use lower camel case when naming variables and functions, and use upper camel case(pascal case) when naming classes.

reference: [Naming Conventions](https://github.com/airbnb/javascript#naming-conventions)